clang-tools-extra/clangd/IncludeCleaner.cpp

```diff
@@ -237,18 +236,6 @@ removeAllUnusedIncludes(llvm::ArrayRef<Diag> UnusedIncludes) {
                            Diag.Fixes.front().Edits.begin(),
                            Diag.Fixes.front().Edits.end());
   }
-
-  // TODO(hokein): emit a suitable text for the label.
-  ChangeAnnotation Annotation = {/*label=*/"",
```

Comment:
(I can't comment on the unchanged code), as we don't use ChangeAnnotation for both cases, I think we can remove the annotation-propagation code (line 271-274) in `fixAll`.